contributors.txt

```diff
@@ -1,5 +1,7 @@
-﻿This is a list of people and organizations that contributed to the NVDA project in various ways since the beginning of the project. For an overview of code contributors, see also
-https://github.com/nvaccess/nvda/graphs/contributors
+﻿This is a list of people and organizations that contributed to the NVDA project in various ways since the beginning of the project.
+For an overview of code contributors, see also https://github.com/nvaccess/nvda/graphs/contributors.
+For a list of active contributors, see the experts list https://github.com/nvaccess/nvda/blob/master/projectDocs/community/expertsList.md
+This list is now archived.
```

Comment:
Should we indicate when this list was archived?  I know it's not up to date now, but saying it was archived in 2024 / 2025 may be useful for anyone looking at the file in, say 10 years time.